styles.css

```diff
@@ -0,0 +1,93 @@
+@media (min-width: 375px) {
```

Comment:
- [ ] Following the best practices for responsive design, your CSS stats off from the small screen and then for bigger screens, you add a media query to properly style for them, :100: :+1:  In that note, at this stage of the project, you are not meant to have any media query yet, your CSS should be able to function for the different mobile screen without restriction, **I'll suggest you remove this media query here, your project will look and adapt great without it** :smile: